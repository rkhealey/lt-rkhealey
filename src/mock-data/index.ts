import profile from './profile.png';

export const user = {
  username: '@yourname',
  imageSrc: profile,
};

export const links = [
  {
    url: 'http://www.google.com',
    text: 'Link One',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Two',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Three',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Four',
  },
  {
    url: 'http://www.google.com',
    text: 'Link Five',
  },
];
