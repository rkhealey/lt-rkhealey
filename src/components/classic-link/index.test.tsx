import React from 'react';
import { shallow } from 'enzyme';
import ClassicLink from '.';

import { links, theme } from '../../mock-data';

describe('Classic Link', () => {
  it('Renders', () => {
    const wrapper = shallow(<ClassicLink {...links[0]} theme={theme} />);
    expect(wrapper).toMatchSnapshot();
  });
});
