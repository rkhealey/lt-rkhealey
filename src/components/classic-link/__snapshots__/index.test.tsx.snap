// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Classic Link Renders 1`] = `
<ForwardRef
  as="a"
  href="http://www.google.com"
  mb={3}
  sx={
    Object {
      ":hover": Object {
        "backgroundColor": "palegoldenrod",
        "color": "rebeccapurple",
      },
      "backgroundColor": "rebeccapurple",
      "color": "palegoldenrod",
    }
  }
  target="_blank"
  variant="block"
>
  Link One
</ForwardRef>
`;
